adding instructions for students (#75)
SanliFaez/OS4Physicists: Mustknow_2023.md

# Open Science for Physicists
*Instructions and required information for academic year 2023-24*

This note contains the specific information for the academic year 2023 for the students of OS4P (ns-ph500m) and OSTP (ns-tp506m) who follow the shared part of the two coruses.

For detailed description of the lectures, check [the Syllabus](Syllabus_202x.md). The exercises for this course are team-based. See [the list of assignments](Exercises_2023.md) for the full list.

> ## Welcome message on Blackboard

Reviewer: Update this with the minor changes I suggested in the email

Lectures 1, 2, 3, and 8 are common with the "Open Science for Theoretical Physics" course (ns-tp506m). The commonalities and differences between these two sister courses will be explained in more details during the first lecture.

The course syllabus and exercises will be posted on the course repository (more information will follow). The crucial announcement will be posted on the Blackboard.

All the lectures are scheduled on Monday mornings from 9:00-12:45, starting on 11 September 2023. 
The venue is [the Learning Plaza](https://www.uu.nl/en/education/future-learning-spaces/learning-spaces/learning-plaza) in the Minnaert building (former canteen).

Sanli Faez, Erik van Sebille, Tanja Hinderer

## What to expect? 

> The OS4P course is about real-life skills and experiences of researchers in a rapidly evolving academic landscape. With the advancement of open science, not only we need an updated look at "how science works", but also need to know which tools are available and which skills will be necessary. Therefore, this course is a combination of lectures and hands-on exercises to get a good feeling of these new developments. 

Reviewer: ```suggestion
The OS4P course is about real-life skills and experiences of researchers in a rapidly evolving academic landscape. With the advancement of open science, students do not only need to update our idea of "how science works", but also need to know which tools are available and which skills will be necessary. Therefore, this course is a combination of lectures and hands-on exercises to get a good feeling of these new developments. 
```

> Some of the open science practices are still not widely embraced by some factions of the scientific community. These frictions will naturally influence the post-graduate career of our students. To appreciate and form an opinion on these practices, the students are sometimes deliberately faced with uncertain circumstances, just like the real world. 

Reviewer: I find 'factions' rather aggressive term. Would the more neutral 'some people in the scientific community' not be better?

Author: adjusted! see next PR

> Some of the open science practices are still not widely embraced by some factions of the scientific community. These frictions will naturally influence the post-graduate career of our students. To appreciate and form an opinion on these practices, the students are sometimes deliberately faced with uncertain circumstances, just like the real world. 

Reviewer: And I fear that the 'sometimes deliberately faced with uncertain circumstances' can be read as scary. Reword?

Author: I extended with this composition:

In this course, we often ask *why* a certain choice is made or commonly practiced. Such questions are not always the most comfortable or easy to answer. We do out best to create a brave environment where such uncomfortable issues can be discussed openly and critically. We hope that all students engage in these discussions with curiosity and an open mind.

Reviewer: Do you really mean 'brave environment'? Not 'safe environment'?

Author: maybe it is a bit woke+ but I am convinced that brave environment is a better expression than safe. This is from a leadership lesson from Brene Brown.

Reviewer: OK is fine; just wanted to make sure it wasn't a type-o ;-)

In this course, we often ask *why* a certain choice is made or practiced. Such questions are not always the most comfortable or easy to answer.  
As the course emphasizes learning from engagement, which is difficult to quantify, grading is done as pass/fail.

## OS4P and OSTP
These two courses are interlinked but follow somewhat different philosophies. OSTP is an adjusted form of the student seminar and put a higher emphasize on research skills and exploring the literature. The OS4P course encompasses a wider skill set that contains setting up measurements, public engagement, science communication, and knowledge transfer from academia to the society. There is also a larger emphasis on good documentation for reproducibility, through the practical example of a measurement project.

> Students are welcome to take each of the courses, or follow both. In the latter case, they will recieve the credits for *only one* of the two courses. 

Reviewer: ```suggestion
Students in the THPH programme are welcome to take each of the courses, or follow both. In the latter case, they will recieve the credits for *only one* of the two courses. 
```

> As the assignments are based on team activities, the OSTP students are kindly asked to either join ALL the 4 extra OS4P lectures if they want to follow this course through.

Reviewer: ```suggestion
As the assignments are based on team activities, the OSTP THPH students that want to also follow OS4P through are kindly asked to join _all_ the 4 extra OS4P lectures.
```

For the final lecture, OSTP students should be aware of the necesary preparation in advance to the lecture.

## What is expected?
To reach a deeper appreciation of the dilemma's in the way of transition to open science, we will organize a lot of group or plenary discussions. Active participation in the tutorials and discussions is more important than getting the most attention and applause for a certain exercise or project. 

> We encourage you stay curious about the topics chosen for these discussions. If somethings does not match your expectation, your are kindly asked to raise those points as soon as you feel comfortable expressing it, in the class, to the student representatives in the focus group, or directly share them with the teachers.

Reviewer: ```suggestion
We encourage you to stay curious about the topics chosen for these discussions. If something does not match your expectation, your are kindly asked to raise those points as soon as you feel comfortable expressing it, in the class, to the student representatives in the focus group, or directly share them with the teachers.
```
